kafka-java-auth/src/main/java/com/google/cloud/hosted/kafka/auth/GcpLoginCallbackHandler.java

```diff
@@ -18,10 +18,15 @@
 
 import static java.nio.charset.StandardCharsets.UTF_8;
 
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdToken;
```

Comment:
Think we need to add `google-api-client` to `kafka-java-auth/pom.xml` 

```
    <dependency>
      <groupId>com.google.api-client</groupId>
      <artifactId>google-api-client</artifactId>
      <version>1.32.1</version>
    </dependency>
```

Reply:
Thanks, added